firebase/src/main/java/org/mozilla/focus/utils/FirebaseNoOpImp.kt

```diff
@@ -12,7 +12,7 @@ import java.util.HashMap
 /**
  * It's a wrapper to communicate with Firebase
  */
-class FirebaseNoOpImp(remoteConfigDefault: HashMap<String, Any> = HashMap()) : FirebaseContract(remoteConfigDefault) {
+open class FirebaseNoOpImp(remoteConfigDefault: HashMap<String, Any> = HashMap()) : FirebaseContract(remoteConfigDefault) {
```

Comment:
why open?

Reply:
since I was wanting to mock this class using `BottomBarRobot.mockBottomBarItems()` and `BottomBarRobot.mockMenuBottomBarItems()`. And I didn't find a good way to make it open and allow to be mock.
Afterward I found that I didn't need to mock the Remote Configs now. But still wanna leave it there for the future usage.